src/App.js

```diff
@@ -1,12 +1,97 @@
 import React from 'react';
+
 import './App.css';
+import PostsList from './components/PostsList/PostsList';
+
+class App extends React.Component{
+  state = {
+    active:0,
+    titleFilt:0,
+    userFilt:0,
+    usersPostsWithComments:[],
+    usersPostsWithCommentsSorted:[]
+  }
+
+  loadDatas = () => {
+    Promise.all([
+     fetch('https://jsonplaceholder.typicode.com/users').then(users => users.json()),
+     fetch('https://jsonplaceholder.typicode.com/posts').then(posts => posts.json()),
+     fetch('https://jsonplaceholder.typicode.com/comments').then(comments => comments.json()),
+ ])
+    .then(data => {
+      const modifiedArray = data[1].map(post => ({
+        ...post,
+        user: data[0].find(user => user.id === post.userId),
+        comments: data[2].filter(comments => comments.postId === post.id),
+      }));
+      this.setState({
+        active:1,
+        usersPostsWithComments: modifiedArray,
+        usersPostsWithCommentsSorted: modifiedArray,
+      })
+    })
+  }
+
+  sortTitle = () => {
+    this.setState(prevState => (
+      this.state.titleFilt === 0
```

Comment:
if you use ternary operator, you have to write the variable you're assigning to you expression